doc/reference/kernel/other/float.rst

```diff
@@ -77,6 +77,19 @@ This means that any thread is allowed to access the floating point registers.
 The ARM kernel automatically detects that a given thread is using the floating
 point registers the first time the thread accesses them.
 
+Pretag a thread that intends to use the FP registers by
```

Comment:
You commented that this pretagging is optional, so you **could** write this instead as:
```
You can optionally pretag a thread that intends ...
```